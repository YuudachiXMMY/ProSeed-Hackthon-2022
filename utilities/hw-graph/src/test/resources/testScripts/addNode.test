# Test to create a graph and and add a node to it.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 node1

# Print all nodes in graph
ListNodes graph1
ListChildren graph1 node1