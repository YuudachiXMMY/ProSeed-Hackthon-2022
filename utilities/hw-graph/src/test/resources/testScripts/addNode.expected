# Test to create a graph and and add a node to it.

# Create a graph
created graph graph1

# Add a node
added node node1 to graph1

# Print all nodes in graph
graph1 contains: node1
the children of node1 in graph1 are: